### PR TITLE
Use home icon as component + change home button color on hover

### PR DESCRIPTION
**Pull request by CPatchane**
The component has been made using SVGR. 
We use a component here to be able to pass a props to use the `currentColor` fill value when needed or the blue color by default.

### CODE REVIEW

**Comment by CPatchane** [comment]
[(𝐒) [BAR] Ajouter un hover sur le nuage](https://trello.com/c/MPiwOzpc/519-%F0%9D%90%92-bar-ajouter-un-hover-sur-le-nuage)

**Comment by CPatchane** [comment]
Have to fix an issue `React does not recognize the `currentColor` prop on a DOM element`

File: package.json [hunk 1]
```diff
@@ -114,9 +113,20 @@
     }
   ],
   "jest": {
-    "moduleFileExtensions": ["js", "jsx", "json", "styl", "yaml"],
-    "setupFiles": ["<rootDir>/test/jestLib/setup.js"],
-    "moduleDirectories": ["node_modules", "src"],
+    "moduleFileExtensions": [
```
Comment (gregorylegarec) [inline package.json]: linting

File: package.json [hunk 2]
```diff
@@ -128,7 +138,9 @@
       "\\.(png|gif|jpe?g|svg)$": "<rootDir>/test/__mocks__/fileMock.js",
       "styles": "identity-obj-proxy"
     },
-    "transformIgnorePatterns": ["node_modules/(?!cozy-ui)"],
+    "transformIgnorePatterns": [
```
Comment (gregorylegarec) [inline package.json]: linting

File: src/components/Apps/IconCozyHome.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+const SvgIconCozyHome = props => {
+  const { currentColor, ...otherProps } = props
+  return (
+    <svg viewBox="0 0 32 32" {...otherProps}>
```
Comment (gregorylegarec) [inline src/components/Apps/IconCozyHome.jsx]: What is the reason to directly use SVG into JSX ?

File: src/styles/apps.css [hunk 4]
```diff
@@ -50,6 +50,16 @@
   margin-right: .75rem;
 }
 
+[role=banner] .coz-nav .coz-nav-apps-btns-home {
```
Comment (gregorylegarec) [inline src/styles/apps.css]: VLS (Very Long Selector)

Is `.coz-nav` absolutely required here as `.cozy-nav-apps-btns-home` is already prefixed with `.cozy-nav` ?

File: package.json [hunk 5]
```diff
@@ -4,7 +4,6 @@
   "description": "cozy-bar.js library, a small lib provided by cozy-stack to inject the Cozy-bar component into each app",
   "main": "dist/cozy-bar.js",
   "author": "Cozy Cloud <contact@cozycloud.cc> (https://cozy.io/)",
-  "contributors": ["m4dz (https://m4dz.net)", "CPatchane"],
```
Comment (gregorylegarec) [inline package.json]: Out of concern of this PR

**Comment by gregorylegarec** [comment]
Can you please provide into the description the reason we are now using SVG directly into the HomeIcon component ? Thanks 🙏

File: src/components/Apps/IconCozyHome.jsx (hunk 3 above)
Comment (CPatchane) [reply src/components/Apps/IconCozyHome.jsx]: We use a component here to be able to pass a props to use the `currentColor` fill value when needed or the blue color by default.

**Pull request merged**